src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="form-search form-big"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label class="form-label form-label-big">
+        <img
```

Comment:
You could use the image as a background for input. So you will have no need to use `img` tag. 
You can try it by yourself if you want.

Reply:
Thank you for a recommendation, I will try!